pom.xml

```diff
@@ -7,12 +7,12 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.backbase.modelbank</groupId>
     <artifactId>engagements-data</artifactId>
-    <version>0.10.2</version>
+    <version>0.11.0</version>
```

Comment:
I'm quite sure you can keep the `0.10.2` version and the pipeline will bump automatically, no?

Reply:
ok. I will keep the 0.10.2